package.json

```diff
@@ -157,7 +157,7 @@
   "bundlesize": [
     {
       "path": "./dist/instantsearch.production.min.js",
-      "maxSize": "70.25 kB"
+      "maxSize": "70.75 kB"
```

Comment:
Could leave a smaller footprint if I could use `find` for the array operations, I could also use `filter` but I don't think it's great cause it iterates on every item